modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -233,12 +231,6 @@ def herbicide_hazard # rubocop:disable Metrics/MethodLength
           end
           @pdf_data[:data][:attributes][:exposureInformation][:toxicExposure][:herbicideHazardService][:serviceDates].delete(:endDate)
         end
-
-        if herb.present?
-          served_in_herbicide_hazard_locations = @pdf_data[:data][:attributes][:toxicExposure][:herbicideHazardService][:servedInHerbicideHazardLocations]
-          @pdf_data[:data][:attributes][:exposureInformation][:toxicExposure][:herbicideHazardService][:servedInHerbicideHazardLocations] =
-            served_in_herbicide_hazard_locations ? 'YES' : nil
-        end
```

Comment:
since we have the check above no we can jsut let this value go through as is if the parent object is present (same with gulfwar)

Reply:
are there any repercussions of case sensitivity here or does our JSON Schemer / Validation enforce the respective casing?

Just curious as I've seen some events with former BGS calls have issues due to casing and it go as a silent failure for a while.

Reply:
@FonzMP   It is type ENUM, so if you send in 'yes' or 'no' it fails to match so we should be good there (below image was created by sending in 'yes' for one and 'no' for the other in place of 'YES' and 'NO')

![Screenshot 2024-05-29 at 10 50 16 AM](https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/07975ad0-cf8f-4629-b456-3ecdab972a88)